frontstage/views/register.py

```diff
@@ -34,6 +35,8 @@ def validate_enrolment_code(enrolment_code):
 
     if result.status_code == 200 and json.loads(result.text)['active']:
         case_id = json.loads(result.text)['caseId']
+    elif result.status_code != 200:
```

Comment:
So it depends on the service, Some services have more specific errors we know they spit out, some have 204's when you look for something that's not found for example.
For any response other than the service specific ones, e.g. 400, 400, 500 we're always just going to be going to a general error page so I feed them all into the ExternalServiceError exception. That will log out the url it was hitting and the status code it returns